IdentityCore/src/MSIDConstants.m

```diff
@@ -37,3 +37,4 @@
 NSString *const MSIDTrustedAuthorityWorldWide    = @"login.microsoftonline.com";
 NSString *const MSIDTrustedAuthorityUSGovernment = @"login-us.microsoftonline.com";
 NSString *const MSIDTrustedAuthorityCloudGovApi  = @"login.usgovcloudapi.net";
+NSString *const MSIDServerErrorClientMismatch    = @"client_mismatch";
```

Comment:
same as before, perhaps should be at MSIDError?

Reply:
It is still a sub error returned by the server. I am not sure on whether we should make it MSIDError.